### ISSUE TITLE
Is there a way to remove question titles?

### ISSUE THREAD

**Issue by BriceonWiley** [priism-center/shinyQuiz#5]
I am considering using shinyquiz multiple times over in an HTML document, but I notice that when I implement each question, I get the header "Practice what you've learned" below the progress bar, and a question name.

Is there capability for this to either be customizable, or full on removable?

**Comment by joemarlo** [comment]
Hmm, that is hardcoded but I agree that it should be customizable or removable. Let me take a look at the codebase and see if there is a quick way to add that to `shinyquiz::set_quiz_options()`.

**Comment by joemarlo** [comment]
I've pushed a new version that includes the ability to modify the heading and turn off the question title. Since this won't go on CRAN for a while, you can install the latest version using `remotes::install_github('priism-center/shinyquiz')`

Here's an example:
```
library(shinyquiz)

quiz <- create_quiz(
  create_question(
    'What is the capital of Illinois?',
    add_choice('Chicago'),
    add_choice('Paw Paw'),
    add_choice('Springfield', correct = TRUE)
  ),
  create_question(
    'Which elements are gases at room temperature? Select all that apply.',
    add_choice('Hydrogen', correct = TRUE),
    add_choice('Mercury'),
    add_choice('Nitrogen', correct = TRUE),
    label = 'Select Hydrogen and Nitrogen'
  ),
  create_question(
    'At what temperature does water boil at sea level?',
    add_slider(min = 90, max = 150, default_position = 120, correct = 100),
    label = 'Select 100'
  ),
  options = set_quiz_options(question_heading = '', include_question_title = FALSE)
)
preview_app(quiz)
```

Please keep in mind I may modify this syntax so there may be breaking changes before going on CRAN.

**Comment by BriceonWiley** [comment]
This worked perfectly! I'll be sure to watch the package so I can monitor any syntax adjustments.

Thanks for your help!

**Comment by BriceonWiley** [comment]
Sorry, I just noticed this after I closed everything. Not a big deal for me, but something to note for maybe someone else. Without the question title visible anymore, the quiz does not give immediate feedback on if an answer is correct, or not. Again, I actually like it this way since it forces the user to wait until the end to see how they did, but I don't know if this was an intentional change, so I wanted to share.
